### PR TITLE
Merge changes from 19.5.0.3 beta (localization testing build)

### PR DESCRIPTION
**Pull request by mokagio**

### CODE REVIEW

File: fastlane/lanes/localization.rb [hunk 1]
```diff
@@ -236,8 +236,10 @@
     release_notes_source = File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources', 'release_notes.txt')
     FileUtils.cp(release_notes_source, File.join(metadata_directory, 'en-US', 'release_notes.txt'))
 
+    metadata_path = File.join(metadata_directory, '**', '*.txt')
+    git_add(path: metadata_path, shell_escape: true)
     git_commit(
-      path: File.join(metadata_directory, '**', '*.txt'),
+      path: metadata_path,
```
Comment (mokagio) [inline fastlane/lanes/localization.rb:242]: See 18f3bf73d1314f58321f3236a3a6a3aabef5be2b for more details on why this changes was necessary.

**Review by mokagio — COMMENTED** [review]
- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress
- [x] Version update in `.xcconfig`
- [x] Diffs from the PRs that made it into this beta – N.A.

File: fastlane/lanes/localization.rb (hunk 1 above)
Comment (AliSoftware) [reply fastlane/lanes/localization.rb:242]: From commit message:
> Maybe this is a hedge case?
🌳 🧳 🦔 😄 

More seriously, the reason `git_commit` fails to add the file if not tracked (even on `allow_nothing_to_commit: true`) is because the underlying git command it runs is `git commit -m message file1 file2 …`, and [as the documentation for git mentions](https://git-scm.com/docs/git-commit/en#:~:text=which%20must%20already%20be%20known%20to%20Git) (emphasis mine):

> The content to be committed can be specified in several ways:
> […]
> 3. by listing files as arguments to the commit command (without --interactive or --patch switch), in which case the commit will ignore changes staged in the index, and instead record the current content of the listed files (**which must already be known to Git**);

So basically it does not add new files, only stage files that are already tracked into git. And since, as you mentioned in your commit message for 18f3bf7, the `release_notes.txt` files for each locale (containing the release notes for previous version) gets _deleted_ during code freeze, hence the error.

Even if this was because `download_metadata.swift` deleted files with no key but we'll soon (🤞 ) migrate to new tooling for that, I still think that this was the right move from the script to delete that file (and thus the right move from you to `git_add` the file back here) if there is no translation for it, including to be sure that we don't accidentally keep the release notes for a previous version around and then accidentally use said previous version with `deliver` on release day if those were still not translated in time.

Comment (AliSoftware) [reply fastlane/lanes/localization.rb:242]: Also, again, worth double-checking — especially at submission time on Friday — that this won't accidentally also add `fastlane/metadata/review_information.txt`, which is a file which contains secrets (i.e. credentials as part of the instructions for Apple to review the app). I highly doubt that it would, because that file is `.gitignore`'d; but, you know, better safe than sorry 🙂

**Review by AliSoftware — APPROVED** [review]
- [X] ~Changes from bugfix PRs which landed in the release branch~
    - None, this was just a beta to ship latest translations ✅ 
 - [X] Version bump in `.xcconfig` files
 - [X] Latest app and metadata translations pulled from Glotpress
    - The whole point of this beta after all 🙂
